video: driver: update SHA# in iris driver recipe

venvenk opened the pull request:
Update the SHA to include the recent fixes made in video driver.

lumag commented on recipes-kernel/iris-video-module/iris-video-dlkm_1.0.bb:
```diff
@@ -4,7 +4,7 @@ LICENSE = "GPL-2.0-only"
 SRC_URI = "git://github.com/qualcomm-linux/video-driver.git;protocol=https;branch=video.qclinux.0.0"
-SRCREV  = "466922b32a5f967176651ace14c38f6874c61b9b"
+SRCREV  = "0278b59a54524ac0d7a06b60bf56395a3104f6e1"
 
```
Ugh. It's Git. So everybody should have the story. Worst case, you should have the story in the branch log. `git log origin/video.qclinux.0.0@{1}` should show you the previous commit that was a part of that branch. You don't need to 'rerecreate' the story. All developers have the story in their git clones.

venvenk commented:
Unfortunately, we do not have local working copy and moreover we updated one commit to update firmware path in downstream driver to pick from upstream.
So, updated with new commit SHA.

ndechesne commented:
I could fetch the commit from Github. It's not in a branch, but it's still in the repo:

`$ git fetch origin 466922b32a5f967176651ace14c38f6874c61b9b
remote: Enumerating objects: 2, done.
remote: Counting objects: 100% (2/2), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 2 (delta 0), reused 2 (delta 0), pack-reused 0 (from 0)
Unpacking objects: 100% (2/2), 1.82 KiB | 1.82 MiB/s, done.
From https://github.com/qualcomm-linux/video-driver
 * branch              466922b32a5f967176651ace14c38f6874c61b9b -> FETCH_HEAD`

vgarodia commented:
video-driver repo is fixed now. This PR can be abandoned. New PR would be raised for firmware path fix.
